src/gatsby/src/components/park/parkHeader.js

```diff
@@ -130,20 +153,7 @@ export default function ParkHeader({
               {isParkOpen !== false &&
                 <p>
                   The {parkType} {parkOperation?.hasParkGate !== false && "gate"} is open {parkDates}
-                  {(parkOperation?.gateOpenTime && parkOperation?.gateCloseTime) ? (
-                    parkOperation.gateOpenTime === "00:00:00" && parkOperation.gateCloseTime === "23:59:00" ? (
-                      <>, 24 hours a day.</>
-                    ) : (
-                      <>
-                        , from <span className="no-wrap">
-                          {parkOperation?.gateOpensAtDawn ? "dawn" : formattedTime(parkOperation.gateOpenTime)}
-                        </span>{" "}
-                        to <span className="no-wrap">
-                          {parkOperation?.gateClosesAtDusk ? "dusk" : formattedTime(parkOperation.gateCloseTime)}
-                        </span>, daily.
-                      </>
-                    )
-                  ) : "."}
+                  {renderGateTimes(parkOperation)}
```

Comment:
Nice, I think that function is a big improvement for readability. One minor thing I notice: there's whitespace here between `{parkDates}` and the function call in the template, so it will render with a space. That's good if it returns "24 hours a day" etc, but if it just returns a `.` you won't want the space before the period. 

We might need to do a conditional `{" "}` in the template to handle that.

Reply:
It doesn't seem to return the space before the period on my end, but let me know if you see it 🙂
![Screenshot 2024-11-12 at 1 26 13 PM](https://github.com/user-attachments/assets/12cc296f-e218-4da7-925c-5e40358c9a27)